deeppavlov/core/common/registry.json

```diff
@@ -25,6 +25,13 @@
   "line_reader": "deeppavlov.dataset_readers.line_reader:LineReader",
   "logit_ranker": "deeppavlov.models.doc_retrieval.logit_ranker:LogitRanker",
   "mask": "deeppavlov.models.preprocessors.mask:Mask",
+  "multitask_input_splitter": "deeppavlov.models.preprocessors.multitask_preprocessor:MultiTaskInputSplitter",
+  "multitask_input_joiner": "deeppavlov.models.preprocessors.multitask_preprocessor:MultiTaskInputJoiner",
```

Comment:
`multitask_input_joiner` doesn't used or mentioned anywhere

Reply:
This is an old artifact. removed